.github/scripts/workflow-commands.functions.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+# https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/workflow-commands-for-github-actions
+
+# Prints the given message to stderr
+function echoerr() {
```

Comment:
rename `echo_err` or `echoErr` for better readability

Reply:
That would involve changing _every_ caller. I can do it… but is that valuable?

Reply:
lets leave it. Its just a nit.